cloud.foundry.cli/src/main/java/cloud/foundry/cli/logic/ApplyLogic.java

```diff
@@ -49,17 +49,29 @@
 
     /**
      * Creates a new instance that will use the provided cf operations internally.
+     * Starts apps automatically by default
```

Comment:
To signal, that this only affects the newly created apps:
```suggestion
     * Starts newly created apps automatically by default
```